tests/test_analog.py

```diff
@@ -292,12 +292,14 @@ def test_hamevo_endianness_cnot() -> None:
     assert torch.allclose(wf_cnot, wf_hamevo, rtol=RTOL, atol=ATOL)
 
 
-@pytest.mark.parametrize("duration", [torch.rand(1), "duration"])
```

Comment:
I thought we allowed duration to be passed within the param_values with @jpmoutinho recent MR?

Reply:
Yes it is possible. This parametrization was actually to cover both the case where you pass it as a value, or as a string and then into the values dict. So maybe it should be kept (or refactored, but still covering both).

Reply:
yes, deleted that and didn't restore - I'll fix that